x-pack/plugins/observability/public/components/shared/exploratory_view/index.tsx

```diff
@@ -66,20 +68,27 @@ export function ExploratoryViewPage({
         ...withNotifyOnErrors(notifications!.toasts),
       });
 
+  const refresh = useCallback(() => setLastRefresh(Date.now()), [setLastRefresh]);
+
   return (
-    <Wrapper>
+    <ObservabilityPageTemplate
+      pageHeader={{
+        pageTitle: TITLE,
+        rightSideItems: [
+          <EuiButton iconType="refresh" onClick={refresh}>
+            {REFRESH_LABEL}
+          </EuiButton>,
```

Comment:
I think this is fine for now, but I'm not sure if the refresh button makes sense here since the chart can be hidden, and then when the user clicks on the refresh button nothing happens, which is a bit weird.

It will be something for https://github.com/elastic/kibana/issues/121999